hledger-lib/Hledger/Data/Journal.hs

```diff
@@ -463,8 +470,8 @@ journalApplyAliases aliases j@Journal{jtxns=ts} =
 -- check balance assertions.
 journalFinalise :: ClockTime -> FilePath -> Text -> Bool -> ParsedJournal -> Either String Journal
 journalFinalise t path txt assrt j@Journal{jfiles=fs} = do
-  (journalNumberAndTieTransactions <$>
-    (journalBalanceTransactions $
+  (journalTieTransactions <$>
```

Comment:
Can you explain more why numbering transactions is no longer needed ? Is it not used for eg the transaction ids in `print -O csv` ?